src/ng-select/themes/ant.design.theme.scss

```diff
@@ -1,5 +1,7 @@
 @import 'mixins';
+@use 'sass:color';
```

Comment:
Move to line 1

Reply:
Why?

Reply:
According to the [sass documentation](https://sass-lang.com/documentation/at-rules/use/), it says: A stylesheet’s @use rules must come before any rules other than @forward, including style rules. However, you can declare variables before @use rules to use when configuring modules.

Otherwise, you will have the compile errors